API.md

```diff
@@ -401,6 +401,55 @@ After first iteration:
 */
 ```
 
+## `client.queryStorageDeal`
+
+> Query a storage deal's status
+
+### `client.queryStorageDeal(cid, [options])`
+
+#### Parameters
+
+| Name | Type | Description |
+|------|------|-------------|
+| cid | `String` | CID of deal to query |
+| options | `Object` | Optional options |
+| options.signal | [`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal) | A signal that can be used to abort the request |
+
+#### Returns
+
+| Type | Description |
+|------|-------------|
+| `Promise<Object>` | Storage deal |
+
+#### Example
+
+```js
+const dealcid = 'zDPWYqFD8CNXu7Mo9qPSUANbTK2vi9vJBnvavF9S1pVGPHafVHpT'
+const storageDeal = await fc.client.queryStorageDeal(dealcid)
+console.log(storageDeal)
+
+/*
+{
+  "State":7,
+  "Message":"",
+  "ProposalCid":
+    {
+      "/":"zDPWYqFD8CNXu7Mo9qPSUANbTK2vi9vJBnvavF9S1pVGPHafVHpT"
+    },
+  "ProofInfo":
+    {
+      "SectorID":1,
+      "CommitmentMessage":
+        {
+          "/":"zDPWYqFCtHkWNkE2p6t6TeV1sPP5kbnKc5ajUhMVV8xvrw1u5F1R"
+        },
+      "PieceInclusionProof":"EiAbbOy4pChsCYqFYA6qJaUJYStlnwYMdQPHZX7YBkVXDD6vgmGTPnWrcdA9M0oAXQCzOq735YKySLUoTI6pAw=="
+    },
+  "Signature":"c2lnbmF0dXJycmVlZQ=="
+}
```

Comment:
To make this more idiomatic for JS users we should convert the root and `ProofInfo` to camel case using the [`toCamel`](https://github.com/filecoin-project/js-filecoin-api-client/blob/master/src/lib/to-camel.js) utility.

e.g. `res.ProofInfo.PieceInclusionProof` should be accessed like `res.proofInfo.pieceInclusionProof`

Reply:
Done!